cluster.go

```diff
@@ -156,10 +156,15 @@ func (c *Cluster) setupConsensus() error {
 		startPeers = peersFromMultiaddrs(c.config.Bootstrap)
 	}
 
+	dataFolder := c.config.ConsensusDataFolder
```

Comment:
Would it make more sense to check this value/change it when loading the config instead?